dao/src/main/java/com/epam/esm/dao/entity/User.java

```diff
@@ -0,0 +1,137 @@
+package com.epam.esm.dao.entity;
+
+import com.epam.esm.dao.entity.audit.AuditListener;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.EntityListeners;
+import javax.persistence.Id;
+import javax.persistence.OneToMany;
+import javax.persistence.Table;
+import java.math.BigDecimal;
+import java.util.HashSet;
+import java.util.Set;
+
+@Entity
+@Table(name = "users")
+@EntityListeners(value = AuditListener.class)
+public class User implements TableEntity {
+  @Id
+  @Column(name = "id")
+  protected Long id;
+
+  @Column(name = "login")
+  private String login;
+
+  @Column(name = "password")
+  private String password;
+
+  @Column(name = "account")
+  private BigDecimal account;
+
+  @OneToMany(mappedBy = "user")
+  private Set<Orders> orders;
```

Comment:
@TimurRadko Why do we need always to get user orders?

Reply:
I know that in the case of a One-to-Many connection, the default download type is LAZY. In this case, related Entity is taken when I refer to it. Maybe I should add visibly declaration this Loading Type?